tree/dataframe/src/RDataFrame.cxx

```diff
@@ -383,6 +389,24 @@ ROOT::EnableImplicitMT();
 ~~~
 Simple as that. More details are given [below](#parallel-execution).
 
+## <a name="collections"></a> Working with collections
+
+RDataFrame reads collections as the special type RVec (e.g. a branch containing an array of floating point numbers can
```

Comment:
Link `RVec` to the respective doxygen entry!

Reply:
Doxygen should do that automatically